src/frontend/efiling-demo/src/components/page/home/Home.js

```diff
@@ -28,7 +28,7 @@ const urlBody = {
       url: "string"
     },
     cancel: {
-      url: "http://localhost:3001/efiling-demo/cancel"
+      url: `${window.location.origin}/efiling-demo/cancel`
```

Comment:
question: doesn't `window.location.origin` return the origin of the page you are currently on? so like since we are on efiling-frontend it will return `localhost:3000` whereas we want to go back to `3001`?

Reply:
This is being set while in the demo, so if the demo is on 3001, it will evaluate to 3001, and send that string to 3000, once on 3000, it is just a string pointing back to 3001